lldb/source/Plugins/Language/Swift/SwiftLanguage.cpp

```diff
@@ -138,6 +138,25 @@ SwiftLanguage::GetMethodNameVariants(ConstString method_name) const {
   if (method_name.GetMangledCounterpart(counterpart))
     if (SwiftLanguageRuntime::IsSwiftMangledName(counterpart.GetStringRef()))
       variant_names.emplace_back(counterpart, eFunctionNameTypeFull);
+
+  // Properties can have multiple accessor blocks. This section of code supports
+  // breakpoints on accessor blocks by name.
```

Comment:
Is accessor "block" a term of art? I would have expected this to be called an "accessor method" or just an "accessor".

Reply:
the documentation of the swift grammar calls them blocks https://docs.swift.org/swift-book/documentation/the-swift-programming-language/summaryofthegrammar#Declarations

I'm fine with removing the word block though.